Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -776,20 +843,23 @@ def tc_get_indicator_command():
                                              headerTransform=pascalToSpace)
         })
 
-
 # @loger
+
+
 def tc_get_indicator(indicator, owners, rating_threshold, confidence_threshold, associated_groups, associated_indicators,
-                     include_observations, include_tags, indicator_type):
+                     include_observations, include_tags, indicator_type, include_attributes):
     raw_indicators = get_indicators(indicator, indicator_type=indicator_type, owners=owners, rating_threshold=rating_threshold,
                                     confidence_threshold=confidence_threshold, associated_groups=associated_groups,
                                     associated_indicators=associated_indicators, include_observations=include_observations,
-                                    include_tags=include_tags)
+                                    include_tags=include_tags, include_attributes=include_attributes)
+
```

Comment:
Extend the command tc_get_indicator to allow for indicator attributes to be extracted